.gitignore

```diff
@@ -0,0 +1,4 @@
+# IDE related
```

Comment:
Let's add the top-level `.idea` directory here as well, which should not be under version control.

Reply:
@errikos Sorry, I wasn't getting notifications for this. 
@LeeTibbert I updated to `1.3.12.` as well